packages/design-system/src/components/button.tsx

```diff
@@ -129,12 +130,13 @@ export const Button = styled("button", {
         color: "$red11",
         "@hover": {
           "&:hover": {
-            boxShadow: "inset 0 0 0 1px $colors$slate8",
+            background: "$red11",
+            color: "$loContrast",
           },
         },
         "&:active": {
-          backgroundColor: "$red3",
-          boxShadow: "inset 0 0 0 1px $colors$red8",
+          backgroundColor: "$red11",
+          color: "$loContrast",
```

Comment:
is this aligned with the button spec from figma for all buttons?

Reply:
For the red one yes